### PR TITLE
Reader/Notifications: add tracks for open post/comment/site

### PR DESCRIPTION
**Pull request by samouri**
tracks added:
- [ ] calyspo_notifications_open_post
- [ ] calypso_notifications_open_comment
- [ ] calypso_notifications_open_site

am i missing anything? is this enough to accomplish what we want @aaronyan ?

### CODE REVIEW

File: client/notifications/index.jsx [hunk 1]
```diff
@@ -159,6 +159,9 @@ export class Notifications extends Component {
 			OPEN_POST: [ ( store, { siteId, postId, href } ) => {
 				if ( config.isEnabled( 'notifications/link-to-reader' ) ) {
 					this.props.checkToggle();
+					analytics.tracks.recordEvent( 'calypso_notifications_open_post', {
+						siteId, postId
+					} );
```
Comment (dmsnell) [inline client/notifications/index.jsx]: whoa! how about them Redux analytics actions dude! let's not reintroduce test-destroying imperative browser-demanding direct network calls here :smile:

**Comment by aaronyan** [comment]
Awesome work @samouri. LGTM! Tested and works well. Only thing we're waiting on is #16887 to land for the links for site-stream to work properly.

**Comment by samouri** [comment]
@dmsnell: is there any reason that Notifications isn't a connected component right now? 

I can wrap it in a `connect` so that it has access to dispatch

**Comment by dmsnell** [comment]
> @dmsnell: is there any reason that Notifications isn't a connected component right now?
> 
> I can wrap it in a connect so that it has access to dispatch

not that I'm aware of - connect away!

File: client/notifications/index.jsx (hunk 1 above)
Comment (samouri) [inline client/notifications/index.jsx]: on it!

**Comment by samouri** [comment]
done. worked in my testing 👍

**Comment by samouri** [comment]
Oops! I mean't to make it "Needs Review" not "Ready to merge"!!!

**Comment by samouri** [comment]
I didn't realize that was possible....but would be cleaner!
I'm sure i'll have to add more stats to notifications sometime soon + i'll make sure to refactor this bit when I do 👍

**Comment by aaronyan** [comment]
LGTM! Thanks @samouri!

**Pull request merged**